pkg/k8shandler/common.go

```diff
@@ -181,15 +181,30 @@ func newElasticsearchContainer(imageName string, envVars []v1.EnvVar, resourceRe
 	}
 }
 
-func newProxyContainer(imageName, clusterName, namespace string, logConfig LogConfig) (v1.Container, error) {
+func newProxyContainer(spec api.ProxySpec, imageName, clusterName, namespace string, logConfig LogConfig) (v1.Container, error) {
 	cpuLimit, err := resource.ParseQuantity("100m")
 	if err != nil {
 		return v1.Container{}, err
 	}
+	if nil != spec.Resources && !spec.Resources.Limits.Cpu().IsZero() {
+		return v1.Container{}, fmt.Errorf("Not allowed to change es-proxy Cpu limit. It is set to %s", cpuLimit.String())
```

Comment:
Do we disallow this elsewhere?

Reply:
i didnt see. 
I added this just a precaution, so that user doesnt misuse it, and i didnt find any way to restrict it in CR.
Suggestions welcome.

Reply:
removed